docs: Added frappe.get_last_doc (#64)
frappe/frappe_docs: frappe_docs/www/docs/user/en/api/document.md

`frappe.get_doc(dict)`

Returns a new Document object in memory which does not exist yet in the
database.
```python
# create a new document
doc = frappe.get_doc({
    'doctype': 'Task',
    'title': 'New Task'
})
doc.insert()
```

`frappe.get_doc(doctype={document_type}, key1 = value1, key2 = value2, ...)`

Returns a new Document object in memory which does not exist yet in the
database.
```python
# create new object with keyword arguments
user = frappe.get_doc(doctype='User', email_id='test@example.com')
user.insert()
```

## frappe.get\_last\_doc
`frappe.get_last_doc(doctype, filters, order_by)`

Returns the last Document object created under the mentioned `doctype`.

```python
# get the last Task created
task = frappe.get_last_doc('Task')
```

> You can also specify filters for to refine your results. For instance, you can
>  retrieve the last cancelled Task by

Reviewer: ```suggestion
You can also specify filters to refine your results. For instance, you can
retrieve the last canceled Task by adding a filter.
```

```python
# get the last available Cancelled Task

> task = frappe.get_last_doc('Task', filters={"Status": "Cancelled"})

Reviewer: ```suggestion
task = frappe.get_last_doc('Task', filters={"status": "Cancelled"})
```

```

> By default, the `order_by` arguement is set to *"creation desc"*, but this value
> can be overridden to use other non-standard fields that can serve the same
> purpose. For instance, you have a field `timestamp` under the **Task** DocType
> that tracks the time it was approved or marked valid instead of the time it was
> created.

Reviewer: ```suggestion
By default, the `order_by` argument is set to `creation desc`, but this value
can be overridden to use other non-standard fields that can serve the same
purpose. For instance, you have a field `timestamp` under the **Task** DocType
that tracks the time it was approved or marked valid instead of the time it was
created.
```
